types/react-redux/react-redux-tests.tsx

```diff
@@ -1208,10 +1214,45 @@ function TestLibraryManagedAttributes() {
     const ConnectedComponent = connect(mapStateToProps)(Component);
     <ConnectedComponent fn={() => { }} />;
 
-    const ConnectedComponent2 = connect<MapStateProps, void, OwnProps>(mapStateToProps)(Component);
+    const ConnectedComponent2 = connect<MapStateProps, void, ExternalOwnProps>(mapStateToProps)(Component);
```

Comment:
It was difficult to keep this behavior because in this PR, we transform the props in the following steps:

- `{ bar: number, fn: () => void, foo: string }`
- → `{ bar?: number | undefined, fn: () => void, foo: string }` (apply JSX.LibraryManagedAttributes)
- → `{ bar?: number | undefined, fn: () => void }` (remove injected props)

`OwnProps` doesn't appear in any of the steps.